kernel/kernel-benchmarks/src/test/java/io/delta/kernel/benchmarks/WorkloadOutputFormat.java

```diff
@@ -349,10 +372,10 @@ public void endRun(Collection<RunResult> result) {
     ObjectMapper mapper = new ObjectMapper();
     try {
       String jsonReport = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(report);
-
-      System.out.println("Generated benchmark report:\n" + jsonReport);
       String outputPath = System.getProperty("user.dir");
-      System.out.println("Writing benchmark report to " + outputPath + "/benchmark_report.json");
```

Comment:
nit: should we save the output path to a final `Path` (and then use .toString to print it)?

this would help ensure the println and the file output remain in sync